tdclient/bulk_import_model.py

```diff
@@ -174,16 +174,17 @@ def upload_part(self, part_name, bytes_or_stream, size):
         self.update()
         return response
 
-    def upload_file(self, part_name, fmt, file_like):
+    def upload_file(self, part_name, fmt, file_like, **kwargs):
```

Comment:
It'd be nice if we would have an example code for dtypes and converters in the docstring or README. Users mainly use this function, but unable to guess how to use `dtypes` or `converters` from this function.

Reply:
I was not sure how much to "copy" the user documentation to docstrings, because I don't know how much the users read the API documentation. But I do like the idea of having the docstring explain things (because I use API documentation!) so I shall do that.

Reply:
I mean, most of the users copy and paste this example for bulk-import https://tdclient.readthedocs.io/en/latest/#bulk-import 

Bulk-import is too complicated to understand how to use it. As you mentioned, it is ok to have an example in README, but I think it'd be helpful to mention possible arguments for `kwargs` proactively, since modern IDE, IPython, or Jupyter easily show docstring.

Reply:
I've added some simpler documentation to the docstrings for the three main import methods (I think those are it?). I shall add something to the README as well.

Reply:
I've added a couple of examples to the README - I've re-used your array example, because I think it makes the point rather well.